integration-cli/docker_cli_pull_local_test.go

```diff
@@ -87,9 +83,7 @@ func (s *DockerRegistrySuite) TestConcurrentPullWholeRepo(c *check.C) {
 	for _, repo := range repos {
 		dockerCmd(c, "inspect", repo)
 		out, _ := dockerCmd(c, "run", "--rm", repo)
-		if strings.TrimSpace(out) != "/bin/sh -c echo "+repo {
-			c.Fatalf("CMD did not contain /bin/sh -c echo %s: %s", repo, out)
-		}
+		c.Assert(strings.TrimSpace(out) != "/bin/sh -c echo "+repo, check.Equals, false, check.Commentf("CMD did not contain /bin/sh -c echo %s: %s", repo, out))
```

Comment:
Either:
```
c.Assert(strings.TrimSpace(out), checker.Equals, expected)
```
or
```
c.Assert(out, checker.Contains, expected)
```

The current logic is hard to follow, and won't give any useful information in a failure case, since it'll show `true != false` or something instead of `<value> != <expected value>`